package.json

```diff
@@ -99,6 +99,8 @@
   "devDependencies": {
     "@babel/core": "^7.8.4",
     "@babel/preset-env": "^7.8.4",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "7.14.5",
+    "@babel/plugin-proposal-optional-chaining": "7.14.5",
```

Comment:
> The JS parser in Webpack 4 doesn't support optional chaining. With targets:"last 2 Chrome versions, last 2 Firefox versions" the code is left untransformed by babel-loader (because those targets support it) so Webpack fails to parse it.

https://stackoverflow.com/questions/62541359/why-do-i-need-babel-plugin-proposal-optional-chaining-in-my-setup-using-bab

Weird we need to add support back in for a thing the base language already supports - oh babel :)

Reply:
yeah it's a bit broken

Reply:
removed these they were not needed